### PR TITLE
Update New-CsTenantNetworkSite.md

### PR DESCRIPTION
**Pull request by hbunkradt**
I have removed the Parameter OnlineVoiceRoutingPolicy, because this parameter is no longer used for location based routing. This parameter still exist on powershell command new-CSNetworkSite, but is no longer used for location based routing. I change example 2 to assign a emergency policy with LBR for a network site

### CODE REVIEW

**Comment by hbunkradt** [comment]
I create a new pull request with changes on syntax and also on example 2

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 1]
```diff
@@ -53,10 +51,10 @@ NetworkSites can exist without all parameters excepts NetworkSiteID. NetworkRegi
 
 ###-------------------------- Example 2 --------------------------
 ```powershell
-PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -OnlineVoiceRoutingPolicy "OVRP1"
+PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -EnableLocationBasedRouting $true -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -EmergencyCallRoutingPolicy "ECRP1"
 ```
 
-The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and with OnlineVoiceRoutingPolicy "OVRP1"
+The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and assign an a emergency policy "ECRP1"
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:57]:
```suggestion
The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and assigns an emergency policy 'ECRP1'.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hbunkradt, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md (hunk 1 above)
Comment (get-itips) [reply skype/skype-ps/skype/New-CsTenantNetworkSite.md:57]: I suggest "assigns a custom emergency call routing policy 'ECRP1'"

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 2]
```diff
@@ -53,10 +51,10 @@ NetworkSites can exist without all parameters excepts NetworkSiteID. NetworkRegi
 
 ###-------------------------- Example 2 --------------------------
 ```powershell
-PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -OnlineVoiceRoutingPolicy "OVRP1"
+PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -EnableLocationBasedRouting $true -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -EmergencyCallRoutingPolicy "ECRP1"
 ```
 
-The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and with OnlineVoiceRoutingPolicy "OVRP1"
+The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and assigns an emergency policy 'ECRP1'.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:57]:
```suggestion
The command shown in Example 2 created the network site 'site2' with description 'site 2'. This site is enabled for LBR, and associates with network region 'RedmondRegion', with location policy 'TestLocationPolicy', and assigns a custom emergency call routing policy 'ECRP1.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@hbunkradt, please commit the suggested change to incorporate @get-itips' suggestion. Many thanks.

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 3]
```diff
@@ -53,10 +51,10 @@ NetworkSites can exist without all parameters excepts NetworkSiteID. NetworkRegi
 
 ###-------------------------- Example 2 --------------------------
 ```powershell
-PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -OnlineVoiceRoutingPolicy "OVRP1"
+PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -EnableLocationBasedRouting $true -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -EmergencyCallRoutingPolicy "ECRP1"
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:54]:
```suggestion
PS C:\> New-CsTenantNetworkSite -NetworkSiteID "site2" -Description "site 2" -NetworkRegionID "RedmondRegion" -LocationPolicy "TestLocationPolicy" -EnableLocationBasedRouting $true -SiteAddress "One Microsoft way" -EmergencyCallRoutingPolicy "ECRP1"
```

Duplicate parameter, please remove

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@hbunkradt Gentle reminder.

**Comment by yogkumgit** [comment]
Closing as there is no response. Please feel free to create a new PR for revised update. Thanks

**Pull request closed without merging**